### PR TITLE
alert fix

### PR DESCRIPTION
**Pull request by theVivek-kumar**
alert done

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    width: 60%;
+    margin: 10px auto;
+    height: auto;
+    padding: 8px;
+    border-radius: 5px;
+    flex-wrap: wrap;
+
+
+
+}
```
Comment (swethaogeti) [inline alert/alert.css:12]: remove extra spacing

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+
+.h2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight: 600;
+    color: rgb(37, 37, 37);
```
Comment (swethaogeti) [inline alert/alert.css:21]: try using var( )

File: alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    background-color: var(--GREEN);
+    margin: 4px;
+    height: 2rem;
+    align-content: center;
+    padding: 5px;
+    border-radius: 5px;
+    color: green;
+    border: 1px solid forestgreen;
+    font-weight: 600;
+    font-size: 18px;
```
Comment (swethaogeti) [inline alert/alert.css:36]: can declare font-size:18px in utility class rather than in  each class

File: alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+}
+
+.alerts-Danges {
+
+    background-color: var(--RED);
+    margin: 4px;
+    height: 2rem;
```
Comment (swethaogeti) [inline alert/alert.css:76]: all the common properties can be declared in  utility class

**Review by swethaogeti — APPROVED** [review]
Good to merge

File: alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
+@import url(/);
```
Comment (Adarsh-Patel-Dev) [inline alert/alert.css:1]: Why your import is empty

File: alert/alert.css [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+.alerts-success {
+
+    background-color: var(--GREEN);
+    margin: 4px;
+    height: 2rem;
+    align-content: center;
+    padding: 5px;
+    border-radius: 5px;
+    color: green;
```
Comment (Adarsh-Patel-Dev) [inline alert/alert.css:33]: Instead of hard coding color declare color variables and use them

File: alert/alert.html [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+<body>
+    <div class="navbar">
+        <div class="navbar-logo">
+            <div class="navbar-logo-title">
+                <h2>KERNEL UI</h2>
+            </div>
+            <div class="navbar-toggle">
+                <div class="navbar-links ">
+                    <a href="#">DOCS</a>
```
Comment (Adarsh-Patel-Dev) [inline alert/alert.html:21]: Add links to respective pages

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**